build.sbt

```diff
@@ -92,12 +89,17 @@ lazy val chiselSettings = Seq (
     else {
       Some("releases" at nexus + "service/local/staging/deploy/maven2")
     }
-  },
+  }
+)
 
-  resolvers ++= Seq(
-    Resolver.sonatypeRepo("snapshots"),
-    Resolver.sonatypeRepo("releases")
-  ),
+val defaultVersions = Map("firrtl" -> "1.1-SNAPSHOT")
+
+lazy val chiselSettings = Seq (
+  name := "chisel3",
+
+  // Provide a managed dependency on X if -DXVersion="" is supplied on the command line.
+  libraryDependencies ++= (Seq("firrtl").map {
+  dep: String => "edu.berkeley.cs" %% dep % sys.props.getOrElse(dep + "Version", defaultVersions(dep)) }),
```

Comment:
Can you also merge the dep into the rest of them? Unless you anticipate dependencies on more than just firrtl, that map really isn't doing much.

Reply:
Right. It was mostly to preserve style with the other repos where the map includes more/all of the BIG4.